DOCS-3084: Update agent.md (#3726)
viamrobotics/docs: docs/configure/agent.md

| `disable_subsystem` | boolean | Optional | When set to `true` it disables the management of `viam-server`. |
| `attributes` | object | Optional | <ul><li>`fast_start`: If set to `true`, `viam-agent` will not wait for a network connection nor check for updates before starting `viam-server`. See [Fast start mode](#fast-start-mode).</li></ul> |

> ### Version updates

Author: ```suggestion
#### Version updates
```

To avoid unexpected downtime when `viam-server` is updated, you can configure a [Maintenance Window](/architecture/viam-server/#maintenance-window).

> With a configured maintennce window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently reconfiguring resources.

Reviewer: ```suggestion
With a configured maintenance window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently reconfiguring resources.
```

> With a configured maintennce window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently reconfiguring resources.

Reviewer: > not currently reconfiguring resources

NetCode uses the word "reconfiguring" all the time, but do users know what this means? Should we say "processing config changes" or something more generic? Up to you was just curious myself.

Author: ```suggestion
With a configured maintenace window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently processing config changes.
```

> With a configured maintenace window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently processing config changes.

Author: ```suggestion
With a configured maintenance window, `viam-agent` will restart and upgrade `viam-server` only when maintenance is allowed and when `viam-server` is not currently processing config changes.
```
